### ISSUE TITLE
allow admin search for licenses based on creation date range

### ISSUE THREAD

**Issue by DrXyzzy** [sagemathinc/cocalc#5030]
This will help with support. Allow creation date range for license query on admin tab license search.

**Comment by williamstein** [comment]
To make the UI simple/clean, would it be enough to just do what Google does and just have a way to restrict to recent cutoffs, e.g., "created in the last week/month/6 months/year" sort of thing?

**Comment by DrXyzzy** [comment]
I don't think that would be enough. Use case is looking for license created when investigating a purchase of a few weeks ago. Stripe shows the timestamp, but often there's no dashboard query that shows the license id. It would be helpful to search within an hour of purchase.

**Comment by DrXyzzy** [comment]
On second thought, a bulk download for admins of all licenses would be an acceptable workaround. It could be as simple as a list of license id's issued since a given start date, or just all license id's. I can use the admin API to get other data & cache it.
